### PR TITLE
EN: Translation of Arkuma slot messages, Storyboard and some tutorial messages

### PR DESCRIPTION
**Pull request by ghost**
Some of the basic translations are a little clunky, so I've tided them up.

### CODE REVIEW

**Comment by Aida-Enna** [comment]
Awesome! It looks good to me, just need the approval from @SynthSy and we'll throw it in 👍 Thanks!

**Review by alama — DISMISSED** [review]
Revert this, if you want to make changes like this, do it in the JP branch and MERGE the changes to all the other branches

**Comment by ghost** [comment]
CI was throwing errors due to the files being in the wrong place. That has now been corrected.

File: Story/Episode 4/pr_043080.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-pr_043080_0010#0,"""<%me>さーん。<br>今どちらにいらっしゃいますか？<br>あ、ゲートエリアにいるみたいですね！"""
+pr_043080_0010#0,"""<%me>さーん。<br>Where are you now?<br>Oh, it looks like you're in the gate area!"""
```
Comment (SynthSy) [inline Story/Episode 4/pr_043080.csv]: Don't do this.

**Review by SynthSy — CHANGES_REQUESTED** [review]

File: Story/Episode 4/pr_043080.csv (hunk 1 above)
Comment (ghost) [reply Story/Episode 4/pr_043080.csv]: I have removed the line breaks. If that's what you mean. If so, then I will make the changes to the other lines.

File: UI/ui_amusement.csv [hunk 2]
```diff
@@ -643,15 +643,15 @@ toast_auto_exit#0,"""一定時間アークマスロットがプレイされな
...
-bubbleStartGame#0,"""準備はよろしいでしょうか？<br>それでは宇宙の果てまで参りましょう。"""
-bubbleRest1#0,"""順調な滑り出しですわ。<br>この調子でお進みくださいませ。"""
-bubbleRest2#0,"""目的地までは、あと１０ステージです。<br>この調子で参りましょう。"""
-bubbleRest3#0,"""目的地まで、残り５ステージですわ。<br>ゴールは目前です、ご武運を。"""
-bubbleHitSpecialPanels#0,"""ダークファルスの反応がございます。<br>ご注意くださいませ。"""
+bubbleStartGame#0,"""Are you ready? Let's go to the edge of the universe."""
+bubbleRest1#0,"""It's going smoothly. Please carry on like this."""
+bubbleRest2#0,"""We have 10 more stages until the destination. Please carry on like this."""
+bubbleRest3#0,"""5 more stages until the destination. You're close to the goal."""
+bubbleHitSpecialPanels#0,"""There's a reaction of Dark Falz. Be careful."""
```
Comment (SynthSy) [inline UI/ui_amusement.csv]: This is more like saying "Dark Falz detected. Watch yourself." or something of that variation.

File: Tutorial/Window/wh_storyboard.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+name01#0,"""About the Storyboard"""
```
Comment (SynthSy) [inline Tutorial/Window/wh_storyboard.csv:1]: Either do the entire file or don't.

File: Story/Episode 4/pr_043080.csv [hunk 4]
```diff
@@ -1,4 +1,4 @@
-pr_043080_0010#0,"""<%me>さーん。<br>今どちらにいらっしゃいますか？<br>あ、ゲートエリアにいるみたいですね！"""
+pr_043080_0010#0,"""<%me>,where are you now? Oh, it looks like you're in the gate area!"""
```
Comment (SynthSy) [inline Story/Episode 4/pr_043080.csv]: This needs fixing.

**Review by SynthSy — CHANGES_REQUESTED** [review]
Aside from the Arkuma Slots, the rest of the PR needs work.

**Comment by ghost** [comment]
The changes have been made.

**Pull request closed without merging**